addon/components/flexberry-maptoolbar.js

```diff
@@ -13,11 +13,20 @@ export default Ember.Component.extend({
 
   activeToolName: null,
 
+  _isMeasureTool(toolname) {
+    return toolname && toolname.substr(-11) ===  'measuretool';
```

Comment:
it is a hack, bad solution. Replace this with property in MapTool class thats indicate what tool can be enabled many times